### PR TITLE
lxde-base/lxpanel: version bump to 0.9.3

### PR DESCRIPTION
**Pull request by charIes17**
Gentoo-Bug: 601586
https://blog.lxde.org/category/lxpanel/

Package-Manager: portage-2.3.0

### CODE REVIEW

File: lxde-base/lxpanel/lxpanel-0.9.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+	use alsa && plugins+=",volumealsa"
+	[[ ${CHOST} == *-interix* ]] && plugins=deskno,kbled,xkb
+
+	econf $(use_enable alsa) --with-x --with-plugins="${plugins}"
+	# the gtk+ dep already pulls in libX11, so we might as well hardcode with-x
+}
+
+src_install () {
+	emake DESTDIR="${D}" install
+	dodoc AUTHORS ChangeLog README
```
Comment (SoapGentoo) [inline lxde-base/lxpanel/lxpanel-0.9.2.ebuild]: remove the last two lines and just use `default`

File: lxde-base/lxpanel/lxpanel-0.9.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+	use wifi && plugins+=",netstat"
+	use alsa && plugins+=",volumealsa"
+	[[ ${CHOST} == *-interix* ]] && plugins=deskno,kbled,xkb
+
+	econf $(use_enable alsa) --with-x --with-plugins="${plugins}"
+	# the gtk+ dep already pulls in libX11, so we might as well hardcode with-x
+}
+
+src_install () {
+	# Get rid of the .la files.
```
Comment (SoapGentoo) [inline lxde-base/lxpanel/lxpanel-0.9.2.ebuild]: I did say replace it with `default`

Comment (charIes17) [inline lxde-base/lxpanel/lxpanel-0.9.2.ebuild]: Sorry, still learning about ebuild writing

**Comment by mrVanDalo** [comment]
@charIes17 I think you have to rebase your commits to only one commit.

**Comment by khumarahn** [comment]
0.9.3 is out

**Pull request closed without merging**